src/handlers/miner_hbbft_handler.erl

```diff
@@ -262,8 +219,11 @@ handle_message(BinMsg, Index, State=#state{hbbft = HBBFT}) ->
                             lager:info("block creation for round ~p took: ~p ms", [NewRound, Duration]),
                             BinTxnsToRemove = [blockchain_txn:serialize(T) || T <- TxnsToRemove],
                             NewerHBBFT = hbbft:finalize_round(NewHBBFT, BinTxnsToRemove),
+                            Buf = hbbft:buf(NewerHBBFT),
```

Comment:
I wonder if this is safe; not all members of the consensus group are guaranteed to reach this point before a new block comes out.